boards/sma_q3/README.md

```diff
@@ -0,0 +1,33 @@
+Platform-Specific Instructions: SMA Q3
+===================================
+
+The SMA Q3 is the smart watch used in [Jazda 2.0](https://jazda.org) and Bangle.js 2.
+It is a platform based around the nRF52840, an SoC with an ARM Cortex-M4 and a BLE radio.
+The smart watch exposes 2 SWD pins, and includes 1 button, 1 backlight LED and a lot of assorted peripherals.
```

Comment:
Is there any sort of datasheet we can link to?

Reply:
The hardware is not open, and the only resource I know about is this writeup by the Bangle.js 2 maker: https://hackaday.io/project/175577-hackable-nrf52840-smart-watch

I don't know how stable hackaday links are for the purpose of including in the docs, but I can add it.

Reply:
Seems better than nothing.

Reply:
Included.